huspacy/tools/custom_code.py

```diff
@@ -0,0 +1,7 @@
+import sys
+sys.path.append("../huspacy/tools")
```

Comment:
Use relative imports instead of this. `__init__.py` should be used instead of this file.